exercises/sum.cpp

```diff
@@ -6,3 +6,20 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+using namespace std;
+
+double sum(double n1, double n2){
+	return n1 + n2;
+}
+
+int main(){
+	double n1, n2;
+	cout<<"Insert the first number: ";
+	cin>>n1;
+	cout<<"Insert the second number: ";
+	cin>>n2;
+	cout<<"Sum: "<<sum(n1,n2);
+	return 0;
```

Comment:
non male 👍 , però metti degli spazi nel codice che scrivi, inoltre muovi in una funzione esterna la lettura dell'input